### ISSUE TITLE
ResNet model pruning

### ISSUE THREAD

**Issue by eeric** [jacobgil/pytorch-pruning#8]
Could it was pruned to ResNet model?

**Comment by jiayouba120035** [comment]
hello, I have some problem with jacob's code, and want to ask you did you implement the code, when I run "python finetune --train",the test accuracy is always about 50%, but the train accuracy is more than 95%, did you have the problem like this, I see you want to prune ResNet, so asking you whether you implement the code.  thank you in advance!

**Comment by eeric** [comment]
ResNet18 model pruned successfully,code in 
https://github.com/eeric/channel_prune
